src/main/java/Application/OpenCVClient/OpenCVClientOperator.java

```diff
@@ -0,0 +1,269 @@
+        } catch (Exception e) {
+            throw e;
+        } finally {
+            ipLock.release();
+        }
+    }
+
+    public void setupClientRunners(String args[]) throws IllegalArgumentException {
+        String[] argument;
+        String clientId = ""; // for test: client13
+        String masterUrl = ""; // for test: http://host.docker.internal:4567/
+        String beatCommand = ""; // for test: client/register
+        String getServerCommand = ""; // for test: client/get_node/
+        processingRunner = new ProcessingRunner();
+
+        int missingParameter = 0;
+        String missingParameterList = "";
+
+        for(int i=0; i<args.length; i++) {
+            System.out.println(args[i]);
+            argument = args[i].split("=");
+            if(argument.length == 2) {
+                switch (argument[0]) {
+                    case "CLIENT_ID":
+                        clientId = argument[1];
+                        break;
+                    case "MASTER_URL":
+                        masterUrl = argument[1];
+                        break;
+                    case "BEAT_COMMAND":
+                        beatCommand = argument[1];
+                        break;
+                    case "GET_SERVER_COMMAND":
+                        getServerCommand = argument[1];
+                        break;
+                    default:
+                        throw new IllegalArgumentException("Invaild argument");
+                }
+            }
+        }
+        if(clientId.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "CLIENT_ID,";
+        }
+        if(masterUrl.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "MASTER_URL,";
+        }
+        if(beatCommand.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "BEAT_COMMAND,";
+        }
+        if(getServerCommand.trim().isEmpty()) {
+            missingParameter++;
+            missingParameterList = missingParameterList + "GET_SERVER_COMMAND";
+        }
+        if(missingParameter > 0) {
+            throw new IllegalArgumentException("Missing Parameter: " + missingParameterList);
+        }
+        String beatUrl = masterUrl + beatCommand;
+        String beatBody =  "{" +
+                "\"id\" : \"" + clientId + "\"" +
+                "}";
+        beatRunner = new HeartBeatRunner(beatUrl, beatBody);
+        String masterCommunicationUrl = masterUrl + getServerCommand + clientId;
+        masterCommunicationRunner = new MasterCommunicationRunner(masterCommunicationUrl);
+    }
+
+    public void detectMarkerInServer() throws JsonProcessingException, InvalidObjectException, IOException {
+        System.out.println("detectMarkerInServer");
+        // connect to server and detect marker
+        String response;
+        JsonNode node;
+
+        startConnection();
```

Comment:
Calling this in a loop (`ProcessingRunner`) creates a new socket everytime. I dont think thats very efficient given that data like IP and port dont change, so you will be creating the same socket every iteration